revolv/templates/project/project.html

```diff
@@ -16,40 +17,123 @@
+            <li class="active tab" id="simple1"> <div class="row">
+            <div class="large-3 push-9 columns" id="impact-and-donate">
+                <div class='large-12 columns' id="donate-or-admin-panel-container">
+                    <div id="donate-box">
+                        <h5><strong>${{ project.amount_donated }}</strong> donated</h5>
+                        <p>of ${{ project.funding_goal }}</p>
+                        <div id="progress-bar">
+                            <div id="green-bar"></div>
+                            <div class="large-4 columns">
+                                <h5 id = "percentage">65%</p>
+                            </div>
+                            <div class="large-8 columns">
+                                <h5 id="days-left">20 DAYS LEFT</p>
+                            </div>
+                        </div>
+                        <br></br>
+                        <div id="button-container">
+                        {% if user.revolvuserprofile.is_ambassador %}
+                        <div id="approve-button"><a href="#">APPROVE</a></div>
+                        <div id="deny-button"><a href="#">DENY</a></div>
+                        <div id="update-button"><a href="#">UPDATE PROJECT</a></div>
+                        {% else %}
+                        <p>Help us reach our goal!</p>
+                        <div id="donate-button"><a href="#donate" data-reveal-id="donate-modal">DONATE</a></div>
+                        {% endif %}
+                        </div>
+                    </div>
+                </div>
+                <div class="large-12 columns" id="impact-stats">
+                    <div id="impact-box">
+                        <h5>Impact</h5>
+                        <p>of {{ project.funding_goal }}</p>
+                        <p>impact information</p>
+                        <p>impact information</p>
+                        <p>impact information<p>
+                    </div>
+                </div>
+            </div>
+            <div class="large-9 pull-3 columns" id="main-project-info">
+                {% if project.video_url %}
+                    <iframe width="700" height="400" id="project_video">
+                    </iframe>
+                {% endif %}
+
+                <h3>{{ project.title }}</h3>
+                <p>Funding Goal: {{ project.funding_goal }}</p>
+                <p>End Date: {{ project.end_date }}</p>
+                <p>Location: {{ project.location}}</p>
+                <p>Organization : {{project.org_name}}</p>
+                <p>About {{project.org_name}} : {{project.org_about}}</p>
+                <p>Mission Statement : {{ project.mission_statement }}</p>
+            </div> 
+            </li>
+            <li class="tab" id="simple2">
+                <br></br>
+                <div class="row">
+                    <div class="large-3 columns">
+                        <div class="update-box">
```

Comment:
Obviously these should be loaded in via the context of this view rather than hardcoded :)